.circleci/config.yml

```diff
@@ -26,7 +26,15 @@ jobs:
 
       - run:
           name: 'Build html'
-          command: python -m sphinx -b html -n -j auto docs docs/_build/html
+          command: sphinx-multiversion docs/ docs/_build/html # python -m sphinx -b html -n -j auto docs docs/_build/html
+
+      - run:
+          name: 'Copy static files'
+          command: cp -r docs/static_landing_page/* docs/_build/html
```

Comment:
Doing these steps manually in the pipeline means it'll look different from doing it locally on my machine, right? Perhaps we should have a script?

Reply:
Since `shutil.copytree()` already exists, I'll look into that.